src/docker.ts

```diff
@@ -79,8 +79,10 @@ const buildAuthForDocker = () => {
   if (dockerUser && dockerPass) {
     logger.debug(`Pulling image as ${dockerUser}`);
     return {
-      username: dockerUser,
-      password: dockerPass
+      authConfig: {
```

Comment:
Based on the code you linked [here](https://lifeomic.slack.com/archives/C01948JLJG7/p1623693030119600?thread_ts=1623686917.104300&cid=C01948JLJG7), I think this should be all-lowercase `authconfig`

Reply:
The one time I didn't CP...